website/docs/release_notes.md

```diff
@@ -5,6 +5,94 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.11](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.11)
+
+### Add support for running multiple snippets of the same kind
+Scala CLI now allows passing and running multiple snippets. Each snippet is then treated as a
+separate input by Scala CLI.
+
+```text
+$ scala-cli --scala-snippet '@main def main() = println(Messages.hello)' --scala-snippet 'object Messages { def hello = "Hello" }'
+Hello
+```
+Added in [#1182](https://github.com/VirtusLab/scala-cli/pull/1182) by [Gedochao](https://github.com/Gedochao)
+
+### Make main class optional in preamble-less assemblies
+It is now allowed to generate an assembly, even for code that has no main class, when `--preamble=false` is passed. This can be useful for libraries, if users want to pass the assembly to tools such as proguard. This also accepts a (hidden) `--main-class-in-manifest=false` option if users want not only no preamble, but also no mention of main class in the assembly manifest (`META-INF/MANIFEST.MF` in the assembly JAR). The latter option is useful for tools, such as the hadoop jar command, that behave differently depending on the presence or not of a main class in the manifest.
+
+Added in [#1200](https://github.com/VirtusLab/scala-cli/pull/1200) by [alexarchambault](https://github.com/alexarchambault)
+
+### Generate scalafmt conf file on fmt command
```

Comment:
IMHO this should be on top

Reply:
by the way, I wouldn't showcase this to users as generating the `.scalafmt.conf` file.
Yes, that's what is happening under the hood, but what really matters to the users is that they can skip creating the configuration file and just pass the version & dialect with scala-cli (or even rely on defaults).

So maybe something like:
```suggestion
### Make `.scalafmt.conf` optional when running the `fmt` sub-command
```
(and change the description accordingly)

Reply:
some example for how the version & dialect can be passed (and how the defaults are inferred) might be a good idea, too.

Reply:
...but yes, we still can mention that generating the configuration (and persisting it in the workspace) is possible with the `--save-scalafmt-conf` option.

Reply:
Moved to top, changed and expanded the description